scripts/dmypy

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
Cloned from scripts/stubgen.

Reply:
You might want to also add a entry_point in setup.py if you want something run from an installed mypy distribution.

Reply:
It's there: https://github.com/python/mypy/pull/4130/files#diff-2eeaed663bd0d25b7e608891384b7298R107

Reply:
Ah! Sorry, I missed that.